### DISCUSSION TITLE
Blazorise license

### DISCUSSION THREAD

**Discussion by nev-21** [abpframework/abp#13071] [Q&A] [answered]
Hi, I just want to know if there is a way to avoid using blazorise in abp community, because its license can't be used in a company (>1million). The company may buy other component library.

**Comment by hikalkan** [comment, marked as answer]
The only way is to rewrite UI layers of the pre-built modules and the theme. BTW, [ABP Commercial](https://commercial.abp.io/) license includes Blazorise license, but it is also commercial :)

Would we consider to change Blazorise. Maybe. But when we started, its license was MIT and completely free. If we switch to another library, it can also be commercial in the future. Nowadays, Blazorise's license model is getting popular and many open source project goes in that direction. All we could do is to be sponsor to Blazorise, so ABP Commercial customers don't pay extra license price for it.

**Reply by nev-21** [reply]
Thanks for your answer @hikalkan

**Reply by fileman** [reply]
Blazorise is interesting but has to many bugs with very slow fix release.
Why not switch to another [free controls library](https://blazor.radzen.com/)
